tests/json/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -241,7 +256,12 @@ trailingComma: "all"
 1e10,1e-3]}}
 
 =====================================output=====================================
-{ key1: [true, false, null], key2: { key3: [1, 2, "3", 1e10, 1e-3] } }
+{
+  key1: [true, false, null],
+  key2: {
+    key3: [1, 2, "3", 1e10, 1e-3],
+  },
+}
```

Comment:
It looks like the object breaking logic is currently being changed even when `--pure` isn't specified...